src/containers/clock.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Chip from '@material-ui/core/Chip';
+import DeleteIcon from '@material-ui/icons/Delete';
+import Fab from '@material-ui/core/Fab';
+import { Typography } from '@material-ui/core';
+import TimelineIcon from '@material-ui/icons/Timeline';
+
+import Stopwatch from '../components/time';
+import sortByTime from '../utils/sort';
+import WatchList from '../components/watch-list';
+
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    minHeight: '100vh',
+    backgroundColor: '#000',
+    textAlign: 'center',
+    flexDirection: 'column',
+  },
+});
+
+const StopwatchControls = ({ onToggle, running}) =>
+  <Fab onClick={onToggle}>
+    {running ? 'Stop' : 'Start'}
+  </Fab>
+
+const bestTime = (attempts) => {
+  if (attempts.length === 0){
+    return '00:00:00';
+  }
+  const best = attempts.sort(sortByTime)[0];
+  return Stopwatch({ ...best, format: 'home' })
+}
+
+const averageTime = (attempts) => {
+  if (attempts.length === 0){
+    return '00:00:00';
+  }
+  const average = attempts.reduce((acc, current) => acc + current.time, 0) / attempts.length;
+  return Stopwatch({ time: average, format: 'home' });
```

Comment:
I would prefer to return the same as the function name instead of react component, because it might make readers confused.

```suggestion
  return attempts.reduce((acc, current) => acc + current.time, 0) / attempts.length;
```